cnd/src/http_api/route_factory.rs

```diff
@@ -20,7 +24,9 @@ pub fn new_action_link(id: &SwapId, action: &str) -> String {
     format!("{}/{}", swap_path(*id), action)
 }
 
-pub fn create<D: MetadataStore + StateStore + AliceSpawner + BobSpawner + Network + Clone>(
+pub fn create<
+    D: MetadataStore + StateStore + SpawnBob + Network + Clone + InitiateRequest + SendRequest,
```

Comment:
I think there are still several places where you use `D` for dependencies but it should actually be `C` for connector.

Reply:
Thanks man, I think I've got them all now.